php-nginx/entrypoint.sh

```diff
@@ -38,6 +38,14 @@ if [ -f ${APP_NGINX_CONF} ]; then
     mv ${APP_NGINX_CONF} ${NGINX_DIR}
 fi
 
+# Configure memcached based session.
+if [ -n ${MEMCACHE_PORT_11211_TCP_ADDR} ] && [ -n ${MEMCACHE_PORT_11211_TCP_PORT} ]; then
```

Comment:
That's fair, although I'm not sure if it's worse to have the extension cause an error or to have it silently fall back to files mode.

Keep them split is fine.